ci/jobs/satellite6-python-trigger.yaml

```diff
@@ -0,0 +1,88 @@
+- job:
+    name: satellite6-pulp-trigger
+    node: f23-vanilla-np
+    wrappers:
+        - ansicolor
+    builders:
+        - shining-panda:
+            build-environment: virtualenv
+            python-version: System-CPython-2.7
+            system-site-packages: true
+            nature: python
+            command: |
+                from __future__ import print_function
+                import logging
+                import os
+                import pip
+                import pprint
+                import sys
+                import time
+                
+                pip.main(['install', 'requests'])
+                import requests
+
+                print('Print out all environment variables...')
```

Comment:
There is no need to print the environment variables since Jenkins already provide a way to inspect it.